packages/cli/test/fixtures/autorun-static-dir/build/folder.html/invesible.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
can you rename this file `invisible.html`? :) just spelling typo

Reply:
if I got a dollar for every typo I make 💃